sympy/stats/crv.py

```diff
@@ -355,7 +359,8 @@ def probability(self, condition, **kwargs):
                 dens = ContinuousDistributionHandmade(dens)
             # Turn problem into univariate case
             space = SingleContinuousPSpace(z, dens)
-            return space.probability(condition.__class__(space.value, 0))
+            result = space.probability(condition.__class__(space.value, 0))
+            return result if not cond_inv else 1 - result
```

Comment:
For consistency with other returns, this should be `S.One - result`, since we are returning a SymPy object. Probably, 1 would be sympified to S.One anyway, but explicit form is better.